README.md

```diff
@@ -1,7 +1,6 @@
 # README
 
-[![Build Status](https://travis-ci.com/slatehorse/npd-find-and-explore.svg?branch=master)](https://travis-ci.com/slatehorse/npd-find-and-explore)
-[![Coverage Status](https://coveralls.io/repos/github/slatehorse/npd-find-and-explore/badge.svg?branch=master)](https://coveralls.io/github/slatehorse/npd-find-and-explore?branch=master)
```

Comment:
Is there a specific reason to remove the coverage status badge?

Reply:
Coveralls isn't enabled within DfE-Digital, so at the moment we don't have a coverage service for the repo.